tabs-tabs-tabs/tabs.js

```diff
@@ -1,23 +1,39 @@
+function firstUnpinnedTab(tabs) {
+  for (var tab of tabs) {
+    if (!tab.pinned) {
+      return tab.index;
+    }
+  }
+}
+
 document.addEventListener("click", function(e) {
   function callOnActiveTab(callback) {
     chrome.tabs.query({}, function(tabs) {
       for (var tab of tabs) {
         if (tab.active) {
-          callback(tab);
+          callback(tab, tabs);
         }
       }
     });
   }
 
   if (e.target.id === "tabs-move-beginning") {
-    callOnActiveTab((tab) => {
-      chrome.tabs.move([tab.id], {index: 0});
+    callOnActiveTab((tab, tabs) => {
+      var destination = 0;
```

Comment:
If you call this `index` rather than `destination`, you can use ES6 shorthand below and get `{index}` rather than `{index: destination}`